src/app/user/user.service.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { TestBed } from '@angular/core/testing';
+
+import { UserService } from './user.service';
+
+describe('UserService', () => {
```

Comment:
Do you think there is any useful testing we should put in place for this component at this time? I don't really see a need yet because the system is so small it is easy to move around and test all of the views manually initially.